### DISCUSSION TITLE
New icon

### DISCUSSION THREAD

**Discussion by Cataclyzor** [relikd/baRSS#10] [Show and tell]
Little refresh to the icon to be more inline with the current MacOS.

![bsRSS](https://user-images.githubusercontent.com/91228758/200272614-ccdbc8be-9275-461f-8d84-3d920648f1f4.png)

**Comment by relikd** [comment]
whups, somehow I missed your post entirely. I like that the style is still simplistic. I'll copy the border to my svg and will include it in a new release. Thank you for the suggestion

**Comment by relikd** [comment]
I've experimented with the new logo but it didnt look right compared to all other apps. I assume the design was different 3 years ago when you posted. Nevertheless, I've shrunk the icon so it will fit the current design.
